sensor_fusion/src/sensor_fusion.h

```diff
@@ -118,7 +125,12 @@ class SensorFusionApplication
 
     bool use_interface_mgr_;
 
-    std::string inertial_frame_name_, body_frame_name_, ned_frame_name_;
+    std::string inertial_frame_name_, body_frame_name_, ned_frame_name_,
+      earth_frame_name_, global_pos_sensor_frame_name_, local_pos_sensor_frame_name_;
+
+    cav_msgs::ExternalObject bsm_obj_;
+
+    long unsigned int last_nav_sat_seq = -1;
```

Comment:
`unsigned` ... `= -1`, just wanted to check that it's intended for this to be MAX_LONG, and possibly architecture dependent?